Plugins/BackgroundFetch/BackgroundFetchPlugin.cs

```diff
@@ -126,7 +126,7 @@ private void RecreateObservable()
                                       string msg;
                                       try
                                       {
-                                          msg = _currentGitUiCommands.GitModule.GitExecutable.GetOutput(args);
+                                          msg = _currentGitUiCommands.GitModule.GitExecutable.Execute(args).AllOutput;
```

Comment:
Anyone with info why stderr is used in this situation?
Generally, either stdout or stderr should be parsed. 
In this case it may be OK as "it is only a message" that is parsed, not data.

Reply:
https://github.com/git/git/commit/96890f4c428ed62cf52c11aba5773d364a8d6234#diff-a8ad3d92e86b1622eb617b0a241f2622911f8e868e3816aebb998a0f3e3bcf95R479

Reply:
https://lore.kernel.org/git/CAH74SFS3hFzYzJ6eb2YenJh4-Pf_VBzVTo8axnEtHDyaJUB-jw@mail.gmail.com/